js/self-starting-sequence/index.ts

```diff
@@ -0,0 +1,19 @@
+import { ReadableApp } from "@scramjet/types";
+import { defer } from "@scramjet/utility";
+
+const app: ReadableApp<string> = async function(_stream,interval:number) {
+    const hub = this.hub!;
+    const instanceId = this.instanceId;
+    const instanceClient = hub.getInstanceClient(instanceId);
+    const seqId = await hub.getInstanceInfo(instanceId).then((res) => { return res.sequence; });
+    const seqClient = hub.getSequenceClient(seqId);
+
+    await defer(interval);
+    this.logger.info(`starting sequence with ${interval} ms interval `);
+    await hub.sendNamedData("info", `starting with id: ${instanceId} +/n`);
```

Comment:
nice but you can write to instance output and set it as topic with given name and content-type